applications/conceptual-model-editor/src/app/diagram/reactflow/components/menu-options.tsx

```diff
@@ -1,12 +1,14 @@
-import { useEffect, useRef, useState } from "react";
+import React, { useEffect, useRef, useState } from "react";
 
 export const useMenuOptions = () => {
     const [isMenuOptionsOpen, setIsMenuOptionsOpen] = useState(false);
     const menuOptionsRef = useRef<HTMLDivElement | null>(null);
 
-    const close = (e: React.MouseEvent | React.FocusEvent) => {
+    const close = (e?: React.MouseEvent | React.FocusEvent) => {
         setIsMenuOptionsOpen(false);
-        e.stopPropagation();
+        if (e !== undefined) {
+            e.stopPropagation();
```

Comment:
It is not clear from the method it self .. that when an event is given it stops the propagation. 
When it is not given it does not stop the propagation.